srcpkgs/devilutionX/files/FindSDL2.cmake

```diff
@@ -0,0 +1,255 @@
+# - Find SDL2
```

Comment:
Where is this file coming from, why is it needed, and why is it not provided by upstream if it is actually needed?

Reply:
The file was present in devilutionX 1.2.1, in later versions removed, with the file, other cpu architectures (example: arm and ppc), can find SDL, without the file, SDL found by i686 and x86_64.

Reply:
In that case, let's not carry the file but fetch it from upstream. See the suggestions below.

Reply:
Done!

Reply:
> other cpu architectures (example: arm and ppc), can find SDL, without the file, SDL found by i686 and x86_64.

I find this statement misleading and possibly false, since I was able to compile this package without the supplied `FindSDL2.cmake`.

Since I see this pull request is going nowhere, may I invite you over to #voidlinux on libera.chat to discuss your work on this package in more detail?

Thank you.

Reply:
If it were false, several distros would not have version 1.2.1 in their repositories yet, and NetBSD uses version 1.2.1, due to problems compiling on various cpu architectures.

PR is making progress yes, Void devs are suggesting improvements and standardizing template.